src/ng/animate.js

```diff
@@ -1,6 +1,117 @@
 'use strict';
 
 var $animateMinErr = minErr('$animate');
+var ELEMENT_NODE = 1;
+
+function mergeClasses(a,b) {
+  if (!a && !b) return '';
+  if (!a) return b;
+  if (!b) return a;
+  if (isArray(a)) a = a.join(' ');
+  if (isArray(b)) b = b.join(' ');
+  return a + ' ' + b;
+}
+
+function extractElementNode(element) {
+  for (var i = 0; i < element.length; i++) {
+    var elm = element[i];
+    if (elm.nodeType === ELEMENT_NODE) {
+      return elm;
+    }
+  }
+}
+
+var $$AnimateClassResolverProvider = [function() {
+  var ADD_CLASS = 1;
+  var REMOVE_CLASS = -1;
+
+  this.$get = [function() {
```

Comment:
You don't need (/ shouldn't ?) use inline array notation when there is nothing to inject (as already mentioned by @petebacondarwin somewhere).
It will not call `toString()` unless the function actually has arguments.